CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # trustdidweb-ts Changelog
 
+## 0.0.2 - 2024-04-04
+
+### Added
+- Add `options` to resolveDID.
+  - Option `versionIdQuery` to query specific version.
```

Comment:
Not sure if these should be standard DID Core query parameters?  versionId and versionTime — https://www.w3.org/TR/did-core/#did-parameters.  Or is there some indirection?